fossevents/base/mail.py

```diff
@@ -0,0 +1,43 @@
+# Third Party Stuff
+from django.conf import settings
+from django.contrib.sites.models import Site
+from django.core.mail import EmailMultiAlternatives
+from django.template import RequestContext, TemplateDoesNotExist
+from django.template.loader import render_to_string
+
+
+def send_email(subject_template_name, email_template_name, to_emails,
+               context=None, html_email_template_name=None, request=None):
+    ctx_dict = {}
+    current_site = Site.objects.get_current()
+    if request is not None:
+        ctx_dict = RequestContext(request, ctx_dict)
```

Comment:
we are already `context=None` there shouldn't be a need to create a new one and expect the request object :)